django-backend/fecfiler/scha_transactions/serializers.py

```diff
@@ -152,110 +148,7 @@ def create_or_update_contact(self, validated_data: dict):
             contact: Contact = Contact.objects.create(**contact_data)
             validated_data["contact_id"] = contact.id
         else:
-            contact = Contact.objects.get(id=tran_contact_id)
```

Comment:
This is SUCH a good cleanup!  I'd recommend renaming `tran_contact_id` to `transaction_contact_id` to remove any ambiguity.